index.js

```diff
@@ -89,4 +88,60 @@ addBtn.addEventListener('click', (e) => {
 });
 
 getData();
-displayBooks();
\ No newline at end of file
+displayBooks();
+
+// Creating Date object for date
+
+// const date = new Date();
+
+// const datePresenter = ()=>{
+//   let month = date.getMonth.toString();
+//   let exactDate = date.getDate();
+//   let hour = date.getHours;
+// }
+// document.body.insertBefore(date, h1);
+// setInterval(datePresenter, 1000);
+
+// Referrence to navbar
+const list = document.querySelector('.list');
+const add = document.querySelector('.add');
+const contact = document.querySelector('.contact');
+const wholeForm = document.querySelector('#form-container');
+const books = document.querySelector('#books-container');
+const body = document.querySelector('body');
+const footer = document.querySelector('footer');
+const contactContainer = document.querySelector('.contact-section');
+
+const listActive = () => {
+  list.classList.add('active-page');
+  add.classList.remove('active-page');
+  contact.classList.remove('active-page');
+  if (document.body.contains(wholeForm)) { body.removeChild(wholeForm); }
+  if (document.body.contains(contactContainer)) { body.removeChild(contactContainer); }
+  body.insertBefore(books, footer);
+  contactContainer.classList.remove('contact-section-active');
+};
+const formActive = () => {
+  list.classList.remove('active-page');
+  add.classList.add('active-page');
+  contact.classList.remove('active-page');
+  if (document.body.contains(books)) { body.removeChild(books); }
+  if (document.body.contains(contactContainer)) { body.removeChild(contactContainer); }
+  body.insertBefore(wholeForm, footer);
+  contactContainer.classList.remove('contact-section-active');
+};
```

Comment:
- [ ] when submitting the form it doesn't do anything, the book doesn't get added to the list.